development/bot_handler.py

```diff
@@ -0,0 +1,32 @@
+import requests
+from environment import get_env_variable
+
+from auth_app.models import Bot
+
+
+SERVER_URL = get_env_variable('SERVER_URL')
+SERVER_PORT = get_env_variable('SERVER_PORT')
+
+
+def get_data_users():
+    try:
+        bots_json = requests.get(
+            '{}:{}/users'.format(SERVER_URL, SERVER_PORT),
+        )
+        data = bots_json.json()
+        users = data['users']
+    except Exception:
+        users = []
+
+    return users
```

Comment:
nit: you can directly return without creating a temp variable like
```
try:
      ....
      return data['users']
except Exception:
      return []

Reply:
done!